node-tests/.eslintrc.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
I'd rather use `overrides` in the the top level `.eslintrc.js` than mkae a new file here...